private/smithy-rpcv2-cbor/src/RpcV2ProtocolClient.ts

```diff
@@ -249,15 +257,16 @@ export class RpcV2ProtocolClient extends __Client<
     let _config_0 = __getRuntimeConfig(configuration || {});
     super(_config_0 as any);
     this.initConfig = _config_0;
-    let _config_1 = resolveCustomEndpointsConfig(_config_0);
```

Comment:
The main point of the change is here. The deprecated `resolveCustomEndpointsConfig` input interface requires a user-supplied endpoint. Equivalent functionality will be provided by the default endpointRuleSet.